src/scenes/CreateEvent/store.js

```diff
@@ -299,7 +290,7 @@ export default class CreateEventStore {
       this.prediction.startTime = nowPlus(TIME_DELAY_FROM_NOW_SEC);
       this.prediction.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + TIME_GAP_MIN_SEC);
       this.resultSetting.startTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + TIME_GAP_MIN_SEC);
-      this.resultSetting.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + (TIME_GAP_MIN_SEC * 2));
+      this.resultSetting.endTime = nowPlus(TIME_DELAY_FROM_NOW_SEC + (TIME_GAP_MIN_SEC * 3));
```

Comment:
the default right now is 48 hours so shouldnt it be times 2?

Reply:
but starttime has one `TIME_GAP_MIN_SEC` from prediction endtime.

Reply:
Ah I see this is for the preset values. Maybe more explicit to set this.prediction.endTime + timeGap *2